doc/double_deposit.md

```diff
@@ -0,0 +1,69 @@
+Similar to ICA, Sending an ibc packet from chain A, include signature of chainB user's signature and execute Tx on chain B according to the signature verification result.
+
+a tx structure will similar to this:
+
+```js
```

Comment:
please use correct language. for example: use `json` here

Reply:
fixed